portfolio/src/main/webapp/script.js

```diff
@@ -33,11 +33,10 @@ function addRandomFunFact() {
 
 var currentProjIndex = 0;
 var projects = ["images/proj1.png", "images/proj2.png", "images/proj3.png", "images/proj4.png"];
-var changeProjectTimeMs = 8000;
+var changeProjectTimeMs = 5000;
+var changeCurrProj;
 
-window.setInterval(function(){
-  changeProject();
-}, changeProjectTimeMs);
+window.onload("changeProject()");
```

Comment:
changeProject should be called periodically right? Can you use setInterval?
https://www.w3schools.com/jsref/met_win_setinterval.asp

Reply:
I switched over from using setInterval to setTimeout because I added functionality that allows users to hover over the image to pause it from changing (so users have enough time to read the descriptions). Since setTimeout only executes the function once after the specified time, it worked well for me.